content/apt/apt-1/contents.lr

```diff
@@ -58,13 +58,33 @@ Warning symptom, when running sudo apt update:
    Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'http://deb.torproject.org/torproject.org focal InRelease' doesn't support architecture 'i386'
    ```
  
-#### 3. Then add the gpg key used to sign the packages by running the following commands at your command prompt
+#### 3. Then import the gpg key used to sign the packages
 
+   You can download the key directly into `/etc/apt/trusted.gpg.d` either 
+   
+   using **cURL**
+   ```
+   sudo curl https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc -o /etc/apt/trusted.gpg.d/torproject.asc
+   ``` 
+   OR using **Wget**
+   
    ```
-   # wget -qO- https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gpg --import
-   # gpg --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | apt-key add -
+   sudo wget https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc -O /etc/apt/trusted.gpg.d/torproject.asc
+
+   ```
+   You can also receive the key from a keyserver using gpg, then export the key into `/etc/apt/trusted.gpg.d` directory:
    ```
+   gpg --keyserver=hkp://keys.gnupg.net --receive-keys 74A941BA219EC810
```

Comment:
This is wrong, we don't using sks public keyservers since we had this issue: https://lists.torproject.org/pipermail/tor-project/2019-June/002378.html